incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"time"
 
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Delete-anchor-crd", func() {
```

Comment:
I think `Delete-anchor-crd` isn't really a good description anymore since we're not _only_ deleting the anchor CRD.

How about `"When deleting CRDs"`, and then the `It()` blocks can have names like `"should not delete subnamespaces"`. This means that when it runs, it will print out something close to "When deleting CRDs, it should not delete subnamespaces" which is exactly what we're testing.

Reply:
done